packages/material-ui/src/LinearProgress/LinearProgress.js

```diff
@@ -228,6 +231,10 @@ LinearProgress.propTypes = {
    * The color of the component. It supports those theme colors that make sense for this component.
    */
   color: PropTypes.oneOf(['primary', 'secondary']),
+  /**
+   * @ignore
+   */
+  styles: PropTypes.object,
```

Comment:
You also need to modify the corresponding type definition in `.d.ts` file.

Reply:
Done. Can you be of any help. Up till now I have not used typescript in React. Can you point me in some right direction. Which would help be a more productive contributor here.

Reply:
@adeelibr I added another change request comment just now.